@ecomplus/storefront-app/src/views/js/Account.js

```diff
@@ -48,7 +48,21 @@ export default {
         this.$router.push({
           name: 'account',
           params: {
-            tab: isShowOrders ? 'orders' : null
+            tab: isShowOrders ? 'orders' : 'favorites'
+          }
+        })
+      }
+    },
+
+    isFavoritesList: {
+      get () {
+        return this.$route.params.tab === 'favorites'
+      },
+      set (isShowFavorites) {
+        this.$router.push({
+          name: 'account',
+          params: {
+            tab: isShowFavorites ? 'favorites' : 'orders'
```

Comment:
perigoso isso dar um loop ainda 😄 
acho que tanto pra tab de pedidos quanto pra tab de favoritos, o unset deveria levar pra rota `account` padrão
```suggestion
            tab: isShowFavorites ? 'favorites' : null
```